data_model/1.3/clusters/FanControl.xml

```diff
@@ -119,36 +119,36 @@ Davis, CA 95616, USA
       </item>
     </enum>
     <enum name="FanModeSequenceEnum">
-      <item value="0" name="Off" summary="Fan is capable of off, low, medium and high modes">
+      <item value="0" name="Off/Low/Med/High" summary="Fan is capable of off, low, medium and high modes">
```

Comment:
Also here, I will check to see what the SDK is calling these and maybe we should consider aligning it. Assuming "/" is not ideal to use in source.

Reply:
Yeah, we adjusted some of these in color control already, should do a spec fix.

Reply:
already fixed in ToT - this is 1.3 https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9369. Going to close this comment - don't think this is something we necessarily need to pull back into 1.3